oarepo_ui/theme/assets/semantic-ui/js/oarepo_ui/forms/components/FormFeedback/FormFeedback.jsx

```diff
@@ -20,21 +20,33 @@ const CustomMessage = ({ children, ...uiProps }) => {
 };
 export const FormFeedback = () => {
   const { values } = useFormikContext();
-  const validationErrors = getIn(values, "validationErrors", {});
+  const BEvalidationErrors = getIn(values, "BEvalidationErrors", {});
+  const FEvalidationErrors = getIn(values, "FEvalidationErrors", {});
   let httpError = getIn(values, "httpErrors", "");
   if (httpError?.response?.data) {
     httpError = httpError?.response?.data.message;
   }
   const successMessage = getIn(values, "successMessage", "");
-  if (!_isEmpty(validationErrors))
+  if (!_isEmpty(BEvalidationErrors))
     return (
       <CustomMessage negative color="orange">
-        <Message.Header>{validationErrors?.errorMessage}</Message.Header>
+        <Message.Header>{BEvalidationErrors?.errorMessage}</Message.Header>
```

Comment:
Can you un-capitalize this please (to avoid confusion with a React component name)?